config/lumi.py

```diff
@@ -49,19 +47,15 @@
                     ],
                     'environs': [
                         'builtin',
-                        'PrgEnv-aocc',
                         'PrgEnv-cray',
                         'PrgEnv-gnu',
-                        'PrgEnv-intel',
-                        'cpeAMD',
+                        'cpeAOCC',
                         'cpeCray',
                         'cpeGNU',
-                        'cpeIntel'
                     ],
-                    'max_jobs': 100,
-                    'modules': ['LUMI'],
```

Comment:
If we do this, the default module (21.08) is loaded and we do not test 21.12. It also lead to failure of the cpeAOCC tests.

Reply:
I think that removing `'modules': ['LUMI']` is not a good idea. To load the `cpeXXX` modules, the `LUMI` module needs to be loaded. ReFrame does that so we don't have to do it ourselves. A solution can be to keep `LUMI` with no version on the configuration and then use the `--map-module LUMI:LUMI/21.12` when we run the tests.

Reply:
make sense. I restored `'modules': ['LUMI']` in the configuration

Reply:
It may be me removing `LUMI` by accident, apologize.